### PR TITLE
Initialize Django-based project using MySQL

### LINKED ISSUE #1: [Backend] 간단한 서버 만들기

### PR DESCRIPTION
**Pull request by utilForever**
This revision includes:
- Initialize Django-based project using MySQL (Resolves #1)
  - Update gitignore to Python project

### CODE REVIEW

File: cafe_list/models.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cafe_address = models.TextField(default= '서울시')           # 주소
+    cafe_floors = models.TextField(default = '1층')            # 규모(층수, 별관)
+    businiess_time = models.TextField(default='매일 영업')      # 운영 시간(요일별로 하면 될듯)
+    complexity = models.TextField(default = '쾌적')             # 혼잡도
+    price_range = models.TextField(default = '저렴')            # 가격대
+    furniture_feature = models.TextField(default = '나무의자')   # 책상 및 의자 특징
+    socket_existence = models.BooleanField(default = True)      #콘센트 유무
+    socket_amount = models.IntegerField(default = '5')          #콘센트 개수(적음/보통/많음/매우많음)
+    wifi_existence = models.BooleanField(default =True)         # 와이파이 유무
+    wifi_speed = models.IntegerField(default = '3')             # 와이파이 속도(느림/보통/쾌적함/빠름) -> 5개로 나눠서 하면 될 듯
```
Comment (onlyshine) [inline cafe_list/models.py:16]: change integer to another data structure

**Review by onlyshine — COMMENTED** [review]
I should change this issue

File: cafe_list/models.py (hunk 1 above)
Comment (utilForever) [reply cafe_list/models.py:16]: You should change this issue. 🗡️

**Review by onlyshine — APPROVED** [review]

**Review by eric98040 — APPROVED** [review]
LGTM

**Pull request merged**